tests/preferences/widgets/test_hamster_combo_box_text.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, unicode_literals
+
+import random
+
+from gi.repository import Gtk
+
+from hamster_gtk.preferences import widgets
+
+
+class TestHamsterComboBoxText(object):
+    """Unittests for HamsterComboBoxText."""
+
+    def test_init(self):
+        """Make sure minimal initialisation works."""
+        combo_box = widgets.HamsterComboBoxText()
+        assert combo_box
+
+    def test_instance(self):
+        """Make sure the widget is still a ComboBoxText."""
+        combo_box = widgets.HamsterComboBoxText()
+        assert isinstance(combo_box, Gtk.ComboBoxText)
+
+    def test_values_constructor(self, combo_box_items):
+        """Make sure the ComboBoxText can be populated via constructor."""
+        combo_box = widgets.HamsterComboBoxText(combo_box_items)
+        assert len(combo_box.get_model()) == len(combo_box_items)
+
+    def test_get_config_value(self, hamster_combo_box_text, combo_box_items):
+        """Make sure the widget value is retrieved correctly."""
+        random_id = random.choice(combo_box_items)[0]
+        hamster_combo_box_text.set_active_id(random_id)
+        assert hamster_combo_box_text.get_config_value() == random_id
+
+        another_random_id = random.choice(combo_box_items)[0]
```

Comment:
Wouldn't it be cleaner to do:
``selected_ids = [item[0] for item in random.sample(combo_box_items), 2)]``
and  then just iterate the test logic over them only once?